fmriprep/run_workflow.py

```diff
@@ -74,6 +67,14 @@ def main():
                          help='use ANTs-based skull-stripping')
 
     opts = parser.parse_args()
+    create_workflow(opts)
+
+
+def create_workflow(opts):
+    import logging
+    from nipype import config as ncfg
+    from fmriprep.workflows import base as fwb
+    from fmriprep.workflows.base import base_workflow_enumerator
```

Comment:
You import inside the function when you want to delay those imports until the function is executed. This can be of interest:

- In niype, when you are wrapping functions in a function node (there's no way to import things otherwise).
- In this case: if you import nipype at the top, when you run ```fmriprep -v``` to see the version you get all the warnings that come with nipype, which is pretty annoying.